Gemfile

```diff
@@ -0,0 +1,9 @@
+source "https://rubygems.org"
+
+# Runtime dependencies are declared honeycomb-opentelemetry.gemspec
+gemspec
+
+# Development dependencies declared here.
+gem "rake", "~> 12.0"
+gem "rspec", "~> 3.0"
+gem "opentelemetry-sdk"
```

Comment:
Should we set a minimum OTel level?

Reply:
Maybe! I was leaving it very open, but some minimums are probably a good idea. I'd like to try to keep it as flexible as possible because the BaggageSpanProcessor at least is not very demanding.

This SDK requirement is only a development (really, test) dependency to get access to a concrete implementation of the abstract stuff defined in the API. So, yea, I'll attempt to find the edges of which parts of the API and the SDK we use and the minimum versions that provide those edges.

Reply:
Added minimum version v1.0.0 (i.e. stable) for API (runtime) and SDK (dev/test) in [f1dae6e](https://github.com/honeycombio/honeycomb-opentelemetry-ruby/pull/5/commits/f1dae6e65b60732a2850ec0f5419c2bf1db3ae7d).